public_dropin_environments/python311_genai/Dockerfile

```diff
@@ -2,6 +2,9 @@
 # It contains a variety of common useful data-science packages and tools.
 FROM datarobot/dropin-env-base-jdk:ubi8.8-py3.11-jdk11.0.22-drum1.11.5-mlops9.2.8
 
+# This makes print statements show up in the logs API
+ENV PYTHONUNBUFFERED=1
```

Comment:
I was right about to do this for all environments. Can I ask you to add this to others as well?

Reply:
Thanks a lot!
I think we can introduce to the listed python-aware base dropins. @yakov-g can you please assist publishing new versions of the base image to our dockerhub?

Reply:
They are in the directory that Erik linked above.
https://github.com/datarobot/datarobot-user-models/tree/master/docker

Reply:
Added, can merge

Reply:
Hold on again.
I'll update drum and image tag in the base envs